interfaces/cython/SConscript

```diff
@@ -122,6 +123,10 @@ else:
     # Install Python module in the default location
     extra = ''
 
+if env['stage_dir']:
+    # Get the absolute path to the stage directory
+    extra += ' --root=' + str(Path.cwd().parents[1].joinpath(env["stage_dir"]))
```

Comment:
This line is long enough to me to deserve its own variable, so it can clarify what is happening here. In particular, an example of what `Path.cwd().parents[1]` _should_ be, in a comment, would help future us remember why this is here. I have a couple of questions as well.

1. Is `Path.cwd()` always guaranteed to be the directory that we want? I mean, `Path.cwd()` returns the current working directory of the Python process, which is not necessarily the root of the project. I think it should be, because SCons requires the SConstruct file, but is there a more stable way to find the right path?
2. `stage_dir` may have relative path components in it, so probably best to `.resolve()` the path after joining.
3. You can avoid the `str(...)` by using an f-string here

Reply:
Regarding (1), SCons changes the working directory such that `cwd()` here is the directory containing the `SConscript` file, the `interfaces/cython` subdirectory of the Cantera source directory. I think that can be regarded as stable behavior.